frontends/web/src/routes/account/send/feetargets.tsx

```diff
@@ -182,7 +182,9 @@ class FeeTargets extends Component<Props, State> {
                                     align="right"
                                     className={`${style.fee} ${style.feeCustom}`}
                                     disabled={disabled}
-                                    label={t('send.fee.label')}
```

Comment:
delete this key from en/app.json?

Reply:
it is also used in various other places i.e. if not custom fee is selected or in the confirming wait-dialog
https://github.com/digitalbitbox/bitbox-wallet-app/search?q=send.fee.label&type=